### ISSUE TITLE
T1B1 dry run - Not possible to close modal via close button or close dropdown selector

### ISSUE THREAD

**Issue by bosomt** [trezor/trezor-suite#15419]
**Describe the bug**
T1B1 dry run - Not possible to close modal via close button or close dropdown selector.
See video, looks like its only possible to click on X when waiting to enter word displayed on device

Follow up of https://github.com/trezor/trezor-suite/pull/15380

**Steps to reproduce:**
1. Start dry run of T1B1
2. Try to close modal via close button
3. Try to close dropdown selector without selecting word

**Info:**
 - Suite version: web 24.12.0 (c4e3d8a47e8ee6e26d264f29f3470336ab0da5c2)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36
 - OS: MacIntel
 - Screen: 1512x982
 - Device: Trezor T1B1 1.12.1 regular (revision 1eb0eb9d91b092e571aac63db4ebff2a07fd8a1f)
 - Transport: WebUsbTransport

**Screenshots:**

https://github.com/user-attachments/assets/23a47ccb-3da8-47a3-a6c7-7218cfc8ffb2

**Comment by Lemonexe** [comment]
There are two separate issues described:
1. Close button does not work: **can't reproduce**, for me it leads to the :warning:  Cancelled state, which is correct :heavy_check_mark:
Tested with T1B1 user env, T1B1 physical and T2B1 user env
2. Select dropdown cannot be closed: **intentional behavior**, as I can see [from code](https://github.com/trezor/trezor-suite/blob/6bb6fd9e59f8c49b12a7e06f5b0ae688d40e550d/packages/suite/src/components/suite/WordInput.tsx#L78) :heavy_check_mark: 
Anywayl I looked if I can remove the option and thus make it closable, but then the Select loses focus after entering a word, which hampers the UX. I don't know if _that's_ correct, but doesn't matter, someone clearly intentionaly made it unclosable :shrug:
